### PR TITLE
Added perftool-testsuite test[V2]

### PR DESCRIPTION
**Pull request by PraveenPenguin**
Added test which run perftool-testsuite and notify
user when test fail

Signed-off-by: Praveen K Pandey <praveen@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by PraveenPenguin** [comment]
job log : https://paste.fedoraproject.org/460202/47730229/

File: perf/perftool.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        tarball = self.fetch_asset("perftool.zip", locations=locations,
+                                   expire='7d')
+        archive.extract(tarball, self.srcdir)
+        self.srcdir = os.path.join(self.srcdir, 'perftool-testsuite-master')
+
+    def test(self):
+        self.count = 0
+        for line in build.run_make(self.srcdir, extra_args='check',
+                                   ignore_status=True).stdout.splitlines():
+            if 'FAIL' in line:
```
Comment (narasimhan-v) [inline perf/perftool.py]: if '-- [ FAIL ] --' in line

Comment (PraveenPenguin) [inline perf/perftool.py]: @narasimhan-v thanks will do

**Comment by PraveenPenguin** [comment]
raised new PR #131

File: perf/perftool.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+from avocado import main
+from avocado.utils import archive, build
+from avocado.utils.software_manager import SoftwareManager
+
+
+class Perftool(Test):
+
+    """
+    perftool-testsuite
+    """
```
Comment (abdhaleegit) [inline perf/perftool.py]: is it good idea to add few lines that explains what this test do ?

**Pull request closed without merging**